README.rst

```diff
@@ -122,12 +122,16 @@ The following `command line arguments`_ can also be used to modify the defaults:
      -l LINE_LENGTH, --line-length LINE_LENGTH
                            How many characters per line to allow [default: 88]
 
+     --diff                Don't write the files back, just output a diff for each file on stdout
+
```

Comment:
I would either use a blank line between every option, or remove the blank line from here, too.

Maybe also keep the same ordering as in the output from `darker --help`, and if it's not logical, fix it in the code as well?

```suggestion
     --diff                Don't write the files back, just output a diff for each file on stdout

```

Reply:
ok.